test/unit/model/history/add-object-change.test.ts

```diff
@@ -51,12 +51,53 @@ describe('AddObjectChange', () => {
     });
   });
   describe('applyChange', () => {
-    // TODO: update this when implimented
-    it('returns Manuscript', () => {
-      const addObjChange = new AddObjectChange('somepath', new ExampleBackmatterClass(), 'someId')
-      const manuscript = {} as unknown as Manuscript;
-      expect(addObjChange.applyChange(manuscript)).toEqual(manuscript);
-    })
+    class SomeObjectClass extends ExampleBackmatterClass {
+      public value1: string = '';
+      public value2: string = '';
+      constructor(val1: string, val2: string) {
+        super();
+        this.value1 = val1;
+        this.value2 = val2;
+      }
+    }
+
+    it('can add object to an empty manuscript array value', () => {
```

Comment:
does it matter if the array is empty or populated?

Reply:
Probably doesn't offer much now but I like to add tests to empty and populated array values just incase theres any `.length` logic added in future.